lib/appmap/class_map.rb

```diff
@@ -82,16 +82,13 @@ def build_from_methods(methods)
       protected
 
       def add_function(root, method)
-        package = method.package
-        static = method.static
-
         object_infos = [
           {
-            name: package.name,
+            name: method.package,
```

Comment:
There are two implementations of `method` now, so the interface has been changed a bit to make that easier.